src/reactapp/src/components/Checkout/Form/GuestEmail.jsx

```diff
@@ -17,7 +21,14 @@ function GuestEmail() {
     setFormToEditMode,
     submitHandler,
   } = useContext(GuestEmailFormContext);
-  const { values } = useFormikContext();
+  const { values, setFieldValue } = useFormikContext();
+  const signInMe = _get(values, fields.signInMe);
```

Comment:
Could we make this variable a bit more logical?
`signInMe` => `showLoginOption` or `customerWantsToSignin`

I've spent a couple of minutes thinking about this, so I understand if this was a hard one to name 😄

Reply:
Alright. let's go for `customerWantsToSignin` here since `showLoginOption` makes sense only when it is used in the component context. However, we need that value on submit where `customerWantsToSignin` makes much more sense.